README.md

```diff
@@ -1,2 +1,196 @@
+
+### Key Features <a name="key-features"></a>
+
+- one of the key features to this website is its responsiveness to mobile and desktop/pc
+- Use the HTML for the structure
+- Use the CSS for the styling
+- Use the design from the Figma 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- https://gisachris.github.io/module-1-capstone-project/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+Example command:
+
+```sh
+ install vs code or any other text editor.
+```
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/gisachris/week1_portfolio```
+
+
+### Install
+
+Install this project with:
+
+
+Example command:
+
+```sh
+  node.js V16 and above
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  use git bash to open in Vs code
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+Example command:
+
+```sh
+Run "npx stylelint "**/*.{css,scss}" --fix " to fix linters 
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+Original design idea by Cindy Shin in Behance.
+-link to original work: 
+https://www.behance.net/adagio07
+
+changes were made by:
+👤 Gisa Chris
+
+-GitHub: https://github.com/gisachris
+
+-Twitter: https://twitter.com/_GisaChris
+
+-LinkedIn: https://www.linkedin.com/in/gisa-chris-907948267/
+
+<span><b>project presentation</b></span>
+
+-https://www.loom.com/share/7be3883887814f309eee71184d86e20d
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+-adding more imagery and completing the portfolio
+-adding tech support 24/7
+- Use the array of objects to display the content
+- Make a contact form for the portfolio
+- Make the desktop version of the design
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page: https://github.com/gisachris/portfolio_week1/issues
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider viewing my other profiles and sending job inquiries. thank you
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my fellow coding partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/gisachris/week1_portfolio/blob/new-features/MIT.md) licensed.
```

Comment:
- [ ] Nice job so far. please you MIT link is broken. kindly fix it
![Screenshot (1004)](https://user-images.githubusercontent.com/65298048/234977364-43d19f4d-2d3e-4a7b-bfe9-597b6f2c623f.png)